examples/mongodb/model.js

```diff
@@ -42,56 +42,65 @@ var OAuthUsersModel = mongoose.model('OAuthUsers');
  */
 
 module.exports.getAccessToken = function(bearerToken) {
-  console.log('in getAccessToken (bearerToken: ' + bearerToken + ')');
```

Comment:
Dropped these in favour of the developer adding `mongoose.set('debug', true);` to the top of the file to get console output.